### PR TITLE
Added first test for classical format

### PR DESCRIPTION
**Pull request by rokdd**
First test for classical format regarding PL #6 . I guess it makes sense to extend them after adding the argument for genre-classical.

### CODE REVIEW

**Comment by Josef-Friedrich** [comment]
The test is ok, but ...

Why did you change the permissions from 755 to 644 of this files?: `_generate-readme.py` and `bin/audiorenamer`.

This files must be executable. Please make these files executable again:

```
chmod a+x _generate-readme.py
chmod a+x bin/audiorenamer
```

`tox -e docs` needs the file `_generate-readme.py` to be executable.

File: test/test_cli_options.py [hunk 1]
```diff
@@ -453,6 +453,26 @@ def test_compilation(self):
...
+
+    def assertDryRun(self, folder, track, test):
+        self.assertEqual(helper.dry_run([
+            '--classical', '--format-classical', '$ar_combined_composer/'
+            '${ar_combined_disctrack}_%shorten{$ar_classical_title,64}_'
+            '%shorten{$acoustid_id,8}',
+            helper.get_testfile('classical', folder, track)
+        ]), test)
+
+    d = '/Debussy_Claude/'
```
Comment (Josef-Friedrich) [inline test/test_cli_options.py:467]: The class attribute `d = '/Debussy_Claude/'` makes only sense if there are several tests. I would write `/Debussy_Claude/01_Pagodes_.mp3`

Comment (rokdd) [reply test/test_cli_options.py:467]: Well I thought for the later tests that might be making sense to keep it but I will take it out.

**Comment by rokdd** [comment]
Because of permissions you are right.. I was wondering and as you noticed I am not yet so experienced with tox :-(

**Comment by Josef-Friedrich** [comment]
Thank you!

**Pull request merged**